scripts/ci/run-ci-tests.sh

```diff
@@ -340,8 +340,14 @@ fi
 # libcriu testing
 make -C test/others/libcriu run
 
-# external namespace testing
-make -C test/others/ns_ext run
+# This depends on the macvlan module which does not exist
+# on the actuated aarch64 runners. If the module cannot
+# be loaded, let's skip the test.
+
+if modprobe macvlan > /dev/null 2>&1 ;then
+	# external namespace testing
+	make -C test/others/ns_ext run
+fi
```

Comment:
Would it be possible to move this check in `test/others/ns_ext/run.sh`?

```diff
--- a/test/others/ns_ext/run.sh
+++ b/test/others/ns_ext/run.sh
@@ -24,6 +24,10 @@ function cleanup()
 
 CRIU=../../../criu/criu
 if [[ "$NS" == "net" ]]; then
+       if ! modprobe macvlan > /dev/null 2>&1 ;then
+               echo "SKIP: macvlan is not supported"
+               exit 0
+       fi
```

Reply:
moved